pallets/msa/src/weights.rs

```diff
@@ -122,6 +122,7 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 			// Standard Error: 6_000
 			.saturating_add(Weight::from_ref_time(1_193_000 as u64).saturating_mul(s as u64))
 			.saturating_add(T::DbWeight::get().reads(2 as u64))
+			.saturating_add(T::DbWeight::get().writes(5 as u64))
```

Comment:
Another weights update I didn't expect.

Reply:
I think these are just older weights, I need to run the benchmarks on this PR